__tests__/webpack-bundle.test.ts

```diff
@@ -32,4 +32,29 @@ describe('webpack-bundle test', () => {
     const result = getCommandOutput(args, folderPath);
     (<any>expect(result)).toMatchSpecificSnapshot(join(folderPath, 'snapshot.js'));
   });
+
+  test('bundle-workflows', () => {
+    const folderPath = join(__dirname, 'webpack-bundle/bundle-workflows');
+    const enryPoint = getEntrypoints(folderPath);
+    const args = getParams('../../../dist/bundle.js', 'bundle', [
+      '--extends=redocly-registry',
+      '--metafile=/tmp/null',
+      '-o=/tmp/null',
+      ...enryPoint,
+    ]);
+    const result = getCommandOutput(args, folderPath);
+    (<any>expect(result)).toMatchSpecificSnapshot(join(folderPath, 'snapshot.js'));
```

Comment:
For bundle we should also check the resulting bundle file (ideally should be a multi-file definition) and generated metafile

Reply:
Yes, this make sense